source/create.js

```diff
@@ -20,7 +20,11 @@ const create = defaults => {
 			options = mergeOptions(defaults.options, options);
 			return defaults.handler(normalizeArguments(url, options, defaults), next);
 		} catch (error) {
-			return Promise.reject(error);
+			if (options.stream) {
+				throw error;
```

Comment:
We could return throwing EventEmitter here. What do you think?

Reply:
No, it's a user-error and should fail immediately.